scripts/watch-less.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+cd src/client/stylesheets
+catw -c 'lessc --include-path=bootstrap -' \
```

Comment:
Is there a particular reason that bootstrap and font-awesome are handled differently for include? If so, it might be good to add a comment about that.